packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -133,138 +67,62 @@ interface IMajorityVoting {
         uint256 votingPower
     );
 
-    /// @notice Emitted when a proposal is created.
-    /// @param proposalId The ID of the proposal.
-    /// @param creator  The creator of the proposal.
-    /// @param metadata The IPFS hash pointing to the proposal metadata.
-    event ProposalCreated(uint256 indexed proposalId, address indexed creator, bytes metadata);
-
-    /// @notice Emitted when a proposal is executed.
-    /// @param proposalId The ID of the proposal.
-    /// @param execResults The bytes array resulting from the proposal execution in the associated DAO.
-    event ProposalExecuted(uint256 indexed proposalId, bytes[] execResults);
+    /// @notice Returns the support threshold parameter stored in the voting settings.
+    /// @return The support threshold parameter.
+    function supportThreshold() external view returns (uint64);
 
-    /// @notice Emitted when the voting settings are updated.
-    /// @param votingMode A parameter to select the vote mode.
-    /// @param supportThreshold The support threshold value.
-    /// @param minParticipation The minimum participation value.
-    /// @param minDuration The minimum duration of the proposal vote in seconds.
-    /// @param minProposerVotingPower The minimum voting power required to create a proposal.
-    event VotingSettingsUpdated(
-        VotingMode votingMode,
-        uint64 supportThreshold,
-        uint64 minParticipation,
-        uint64 minDuration,
-        uint256 minProposerVotingPower
-    );
```

Comment:
Why are these events and structs removed from here? Tally feels like a voting struct used to define types and results (interface), not just contract-scoped

Reply:
> Why are these events and structs removed from here?

In your task description you wrote:

> Split the MajorityVoting interface into two:
> - Proposal interface
>     - createProposal + event
>     - executeProposal + event   (default empty implementation)
> - MajorityVoting interface
>     - vote + event   (default empty implementation)
>     - supportThreshold
>     - minParticipation
>     - canExecute
>     - canVote

Are you saying these events should have stayed inside `IMajorityVoting`? This is where they were located before I started with this task.

Initially, I moved `event ProposalCreated` and `event ProposalExecuted` into a separate `interface IProposal` (and created abstract `GovernancePlugin...` contracts to facilitate emitting these events). We talked about it in a 1 on 1.
Then, as an outcome of https://github.com/aragon/core/pull/167#discussion_r1054898602 and the call I had with the rest of the team yesterday, I moved it into the specializations (`AddresslistVoting`, `TokenVoting`, `Multisig`, `Admin`).

> Tally feels like a voting struct used to define types and results (interface), not just contract-scoped
Before this task, the `Tally` struct was part of `IMajorityVoting` because I had put it there. Because of the task description that didn't mention the tally results in the interface description, it seemed to me that it should be moved elsewhere.

Reply:
Makes sense, feel free to resolve this comment :+1: